gwcs/wcs.py

```diff
@@ -634,3 +641,273 @@ def fix_inputs(self, fixed):
         new_pipeline.append((step0[0], new_transform))
         new_pipeline.extend(self.pipeline[1:])
         return self.__class__(new_pipeline)
+
+    def to_fits_sip(self, bounding_box, max_pix_error=0.25, degree=None,
+                    max_inv_pix_error=0.25, inv_degree=None, 
+                    npoints=32, verbose=False):
+        """
+        Construct a SIP-based approximation to the WCS in the form of a FITS header
+
+        This assumes a tangent projection.
+
+        The default mode in using this attempts to achieve roughly 0.25 pixel
+        accuracy over the whole image.
+
+        Parameters
+        ----------
+        bounding_box : a pair of tuples, each consisting of two integers
```

Comment:
```suggestion
        bounding_box : a pair of tuples, each consisting of two integers, optional
```

Reply:
Not optional. I need that and gwcs isn't guaranteed to have it. If it was stored in some standard meta data item, I could see if it existed.

Reply:
`optional` is used to indicate that a specific function argument has a `default` value assigned to it in the function declaration.

Reply:
Yeah, but where is that default value in the code? I don't see it in mine, but maybe github solved that problem for me ;-) Going for a walk...

Reply:
You are right: I missed that this one does not have a default value.